src/applications/avs/tests/e2e/avs.cypress.spec.js

```diff
@@ -24,7 +27,17 @@ const setup = ({
   cy.intercept('GET', `/avs/v0/avs/*`, {
     statusCode: apiStatus,
     delay: avsDelay,
-    body: apiStatus === 200 ? avsData : {},
+    body:
+      // eslint-disable-next-line no-nested-ternary
+      apiStatus === 200
```

Comment:
Maybe it makes sense to use a function and a series of `if..return` clauses to handle this instead of a nested ternary? I'll leave that up to you.